sentry_streams/sentry_streams/examples/transformer.py

```diff
@@ -38,12 +39,13 @@ def __init__(self) -> None:
         self.batch: MutableSequence[Any] = []
 
     def add(self, value: Any) -> Self:
```

Comment:
Message[] will need to be become part of the type annotations within Accumulator class definition, etc. I'm working on it